Find a functionally equivalent code:com.android.internal.policy.impl.PhoneWindow.onIntChanged:COMMENT
Method Modifier: protected   internal    
Comment:/**
 * Called when an int feature changes, for the window to update its
 * graphics.
 *
 * @param featureId The feature being changed.
 * @param value The new integer value.
 */

@@ -2,8 +2,8 @@
     if (featureId == FEATURE_PROGRESS || featureId == FEATURE_INDETERMINATE_PROGRESS) {
         updateProgressBars(value);
     } else if (featureId == FEATURE_CUSTOM_TITLE) {
-        FrameLayout titleContainer = (FrameLayout) findViewById(com.android.internal.R.id.title_container);
+        FrameLayout titleContainer = (FrameLayout) findViewById(R.id.title_container);
         if (titleContainer != null) {
             mLayoutInflater.inflate(value, titleContainer);
         }

